criu/arch/aarch64/crtools.c

```diff
@@ -23,6 +23,37 @@
 #include "compel/infect.h"
 #include "pstree.h"
 
+#ifndef HWCAP_PACA
+/* Copied from linux: arch/arm64/include/uapi/asm/hwcap.h */
+#define HWCAP_PACA (1 << 30)
+
+/* Copied from linux: arch/arm64/include/uapi/asm/ptrace.h */
+/* pointer authentication keys (NT_ARM_PACA_KEYS, NT_ARM_PACG_KEYS) */
+struct user_pac_address_keys {
```

Comment:
These structures and HWCAP constants are defined in different headers. I think it is better to declare cr_user_pac_address_keys that will be a copy of the uapi user_pac_address_keys.

Reply:
This is a good idea. Thanks Andrei!